cfme/tests/services/test_dialog_element_in_catalog.py

```diff
@@ -908,7 +912,21 @@ def test_service_dynamic_dialog_tagcontrol():
             4.
             5. Tag control drop down should show correct values
     """
-    pass
+    catalog_item, sd, ele_label = generic_catalog_item_with_imported_dialog
+    service_catalogs = ServiceCatalogs(appliance, catalog_item.catalog, catalog_item.name)
+    view = navigate_to(service_catalogs, "Order")
+
+    view.fields('tag_control_1').dropdown.fill("London")
+    wait_for(
+        lambda: view.fields("tag_control_1").read() == "London" and
+        view.fields('text_box_1').read() == "Tag: 'London'", timeout=7
+    )
+
+    view.fields('tag_control_1').dropdown.fill("New York")
+    wait_for(
+        lambda: view.fields("tag_control_1").read() == "New York" and
+        view.fields('text_box_1').read() == "Tag: 'New York'", timeout=7
+    )
```

Comment:
**Optional:** It will helpful if you add comment for these strings - ```"Tag: 'New York'"```, ```"tag_control_1"```, ```"New York"```, ```Tag: 'London'``` and others